### ISSUE TITLE
100% opacity?

### ISSUE THREAD

**Issue by Malmsten94** [simon-xia/atom-global-background#6]
Hi!

Would like the background to be 100% and then the text 100% over it. I'm guessing the img is an overlay now. Can it be placed between the text and standard background? 👍 :)

**Comment by TiriaAndersen** [comment]
Yeah, when I increase background opacity it fades out the text, making settings & editor code a lot harder to see. 

Is there any way to fix this?
